frontends/common/applyOptionsPragmas.h

```diff
@@ -76,9 +76,13 @@ class ApplyOptionsPragmas : public Inspector {
  * they're unable to parse a pragma.
  */
 class P4COptionPragmaParser : public IOptionPragmaParser {
+    bool supportCommandLinePragma;
+
  public:
     std::optional<CommandLineOptions> tryToParse(const IR::Annotation *annotation) override;
 
+    explicit P4COptionPragmaParser(bool sCLP) : supportCommandLinePragma(sCLP) {}
```

Comment:
nitpick: I would just spell fully the long name for the sake of editors that use the signature to help the programmer with calling the constructor.